lasagne/tests/layers/test_noise.py

```diff
@@ -138,7 +138,7 @@ def test_get_output_for_deterministic(self, layer):
         input = theano.shared(numpy.ones((3, 3)))
         result = layer.get_output_for(input, deterministic=True)
         result_eval = result.eval()
-        assert (result_eval == input.eval()).all()
+        assert (np.array(result_eval) == np.array(input.eval())).all()
```

Comment:
I did actually check the ```np.allclose``` and it was true. So I was really surprised at first that the ```==``` gave ```false```.  I think changing all the tests could be another PR but for now I will change this one to ```np.allclose``` and test it with float64 and float32.